openshift/04-deployment-template.yml

```diff
@@ -77,10 +77,12 @@ objects:
           - name: THREESCALE_DEPLOYMENT_ENV
             value: "${ENVIRONMENT}"
           - name: THREESCALE_PORTAL_ENDPOINT
-            value: "http://apicast-mapping-service/config"
-          image: "${APICAST_IMAGE}"
+            value: "http://apicast-mapping-service-${RELEASE_REF}/config"
+          - name: APICAST_OIDC_LOG_LEVEL
+            value: "${APICAST_OIDC_LOG_LEVEL}"
+          image: apicast-cloud-hosted:${RELEASE_REF}
           imagePullPolicy: IfNotPresent
-          name: apicast
+          name: apicast-${RELEASE_REF}
```

Comment:
There should be no need for containers to have a suffix no?

Reply:
yes sure